README.md

```diff
@@ -59,6 +59,70 @@ f1.content = "The quick brown fox jumped over the lazy dog."
 bf1.save
 ```
 
+## Extending Directly Contained Derivatives
```

Comment:
For simplicity's sake, can we name this heading something like "Custom Derivatives"? While I appreciate the specificity around how the derivs will be contained (in LDP parlance), I suspect many users of Hydra::Works may not have much knowledge of the intricacies of LDP.